presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -154,6 +155,11 @@ public SystemSessionProperties(
                             return concurrency;
                         },
                         value -> value),
+                integerSessionProperty(
+                        LOCAL_EXCHANGE_CONCURRENCY,
+                        "Number of local parallel hash partitioners per local exchange",
+                        1,
```

Comment:
Is the default set 1 to reduce the risk of this change? If so, we should test it better to make sure it's safe (wrt concurrency issues). If not, then this should be set to a number that maximizes performance (i.e. we need benchmarks at 3-4 different settings)